### ISSUE TITLE
Error: Page is out of bounds

### ISSUE THREAD

**Issue by PythonCoderAS** [HealthScreening/HealthScreeningBot#198]
```
/Users/aoyansarkar/WebstormProjects/HealthScreeningBot/src/utils/paginator.js:68
            throw Error("Page is out of bounds!");
                  ^

Error: Page is out of bounds!
    at Paginator.setPage (/Users/aoyansarkar/WebstormProjects/HealthScreeningBot/src/utils/paginator.js:68:19)
    at Object.collector (/Users/aoyansarkar/WebstormProjects/HealthScreeningBot/src/utils/paginator.js:83:28)
    at InteractionCollector.<anonymous> (/Users/aoyansarkar/WebstormProjects/HealthScreeningBot/src/utils/customCollector.js:74:29)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)

Node.js v17.2.0
```

**Comment by PythonCoderAS** [comment]
Duplicate of #221

**Comment by PythonCoderAS** [comment]
Fixed by #225
